src/index.js

```diff
@@ -117,7 +125,9 @@ if (typeof INSTALL_PRODUCT === 'undefined') {
   INSTALL_PRODUCT = undefined;
 }
 
-urlPatterns = urlPatterns.map(({ applicationId, regex }) => {
+let appIdUrlRegexArr = urlPatterns
+.filter(x => x && (x.regex || x.applicationId)) // appId / urlRegEx both empty
+.map(({ regex, applicationId }) => {
```

Comment:
you changed the order?

Reply:
looks like variable names are strict, meaning they can only be keys in hash, hence the order doesn’t matter.